core/src/main/java/nl/nn/adapterframework/extensions/svn/SvnUtils.java

```diff
@@ -77,9 +78,8 @@ private static String getHeadHtml(String urlString) throws ConfigurationExceptio
 	}
 
 	private static String getReportHtml(String urlString, String revision, String path) throws ConfigurationException, SenderException, TimeoutException, IOException {
-		HttpSender httpSender = null;
+		HttpSender httpSender = new HttpSender();
```

Comment:
In the name of consistent code, can you please also make this same change on line 61 of method `getHeadHtml()`? And remove the null-check from line 74.

Reply:
Ah yes, of course.